docs/setup/embedding.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[embedding]]
+== Embedding {kib}
+
+You can embed a {kib} dashboard or visualization using an HTML code snippet generated with the *Copy iFrame code* button in *Share > Embed code* menu in *Dashboard* or *Visualize*.
+
+NOTE: Embedding of any other part of {kib} is also generally possible, but you may need to craft the proper HTML code manually.
+
+[role="screenshot"]
+image::images/embed-kibana.png[Generate an HTML snippet to embed {kib}, align=center]
+
+[float]
+[[embedding-security]]
+=== Configure security
+
+Embedding through iframes, if used properly, does not directly pose a security risk, but still requires a careful consideration.
+
+[float]
+==== Authentication
+If you're embedding {kib} to the website that supports Single Sign-On with SAML, OpenID Connect, Kerberos or PKI it's highly advisable to configure {kib} as a part of this Single Sign-On setup as well. Operating within a single and properly configured security domain would provide you with the most secure and seamless user experience. You can read more at <<kibana-authentication, Authentication in {kib}>>.
+
+If you want anyone to access embedded {kib} skipping the login step, but the Single Sign-On isn't an option for you, consider configuring the <<anonymous-authentication, anonymous access>> instead. It is already natively integrated into the embedding workflow for the dashboards and visualizations.
```

Comment:
```suggestion
If you want users to access embedded {kib} by skipping the login step, and Single Sign-On isn't an option for you, consider configuring <<anonymous-authentication, anonymous access>>. It is already natively integrated into the embedded workflow for the dashboards and visualizations.
```

Reply:
> It is already natively integrated into the **embedded workflow** for the dashboards and visualizations.

I was trying to say `the workflow for embedding of the dashboards and visualizations`, @gchaps would `embedded workflow` be the right expression here? To my non-native ear it sounds like a workflow is embedded into something :see_no_evil:

Reply:
I like how you said it here. Does this work?

> It is already natively integrated into the workflow for embedding dashboards and visualizations.

Reply:
Yep, this works, thanks!